index.html

```diff
@@ -23,204 +30,207 @@
+          <p class="workText2 Interfont">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul class="tecnoBTN">
+            <li>html</li>
+            <li>css</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button type="button" class="PrjBTN SePrjBTN Interfont BTN-transition">
+            See Project
+          </button>
+        </div>
+      </div>
+      
     </section>
-    <!-- Artwork 1 -->
-    <div class="ArtData worksection multiPost">
-      <h3 class="Txt ArtTxt textcolor workText1 CreteRoundfont">
-        Profesional Art Printing Data
-      </h3>
-      <p class="Txt textcolor workText2 Interfont">
-        A daily selection of privately personalized reads; no accounts or
-        sign-ups required. has been the industry's standard
-      </p>
-      <ul class="ARTtecnoBTN">
-        <li>html</li>
-        <li>bootstrap</li>
-        <li>Ruby</li>
-      </ul>
-      <button type="button" class="PrjBTN Interfont">See Project</button>
-    </div>
-    <!-- Artwork 2 -->
-    <div class="ArtData worksection multiPost">
-      <h3 class="Txt ArtTxt textcolor workText1 CreteRoundfont">
-        Profesional Art Printing Data
-      </h3>
-      <p class="Txt textcolor workText2 Interfont">
-        A daily selection of privately personalized reads; no accounts or
-        sign-ups required. has been the industry's standard
-      </p>
-      <ul class="ARTtecnoBTN">
-        <li>html</li>
-        <li>bootstrap</li>
-        <li>Ruby</li>
-      </ul>
-      <button type="button" class="PrjBTN Interfont">See Project</button>
-    </div>
-    <!-- Artwork 3 -->
-    <div class="ArtData worksection multiPost">
-      <h3 class="Txt ArtTxt textcolor workText1 CreteRoundfont">
-        Profesional Art Printing Data
-      </h3>
-      <p class="Txt textcolor workText2 Interfont">
-        A daily selection of privately personalized reads; no accounts or
-        sign-ups required. has been the industry's standard
-      </p>
-      <ul class="ARTtecnoBTN">
-        <li>html</li>
-        <li>bootstrap</li>
-        <li>Ruby</li>
-      </ul>
-      <button type="button" class="PrjBTN Interfont">See Project</button>
-    </div>
-    <!-- Artwork 4 -->
-    <div class="ArtData worksection multiPost">
-      <h3 class="Txt ArtTxt textcolor workText1 CreteRoundfont">
-        Profesional Art Printing Data
-      </h3>
-      <p class="Txt textcolor workText2 Interfont">
-        A daily selection of privately personalized reads; no accounts or
-        sign-ups required. has been the industry's standard
-      </p>
-      <ul class="ARTtecnoBTN">
-        <li>html</li>
-        <li>bootstrap</li>
-        <li>Ruby</li>
-      </ul>
```

Comment:
**[REQUIRED CHANGES]**

- [ ] Hey, as required by the previous review, Please kindly align your list items to the left to be on same line with the paragraph and header content below: 

**YOUR DESIGN** :point_down: 
![align](https://user-images.githubusercontent.com/59397159/167247650-6c863229-0e79-4bd5-9275-a3bbdba38ce1.png)

**FIGMA DESIGN** :point_down: 


![figma](https://user-images.githubusercontent.com/59397159/167247718-79d97568-21f9-440a-a73b-ba093edd84d9.png)